index.html

```diff
@@ -8,20 +8,40 @@
     <link rel="stylesheet" href="style.css" />
     <title>Awsome Books</title>
   </head>
-  <body class="background-image">
-    <main>
-      <section class="hero is-medium">
+  <body class="background-image" id="body">
+    <header class="is-dark">
+      <nav class="navbar is-dark h-p">
+        <div class="container">
+          <div class="navbar-brand">
+            <a class="navbar-item is-size-4" href="#" id="index"> Awesome Book Store </a>
+            <a role="button" class="navbar-burger" data-target="navbarBasicExample">
+              <span aria-hidden="true"></span>
+              <span aria-hidden="true"></span>
+              <span aria-hidden="true"></span>
+            </a>
+          </div>
+          <div id="navbarBasicExample" class="navbar-menu">
+            <div class="navbar-end">
+              <a href="#all-books" class="navbar-item is-size-6" id="all-books"> All Books </a>
+              <a href="#add-book" class="navbar-item is-size-6" id="add-book"> Add new book </a>
+              <a href="#contact-us" class="navbar-item is-size-6" id="contact-us"> Contact Us </a>
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main class="verticle-align">
+      <section class="hero is-medium" id="allbooks">
         <div class="hero-body">
           <div class="container has-text-centered">
             <h1 id="main" class="main-title">All Awesome Books</h1>
-            <!-- <ul class="book-list sub-title"></ul> -->
             <aside class="menu">
               <ul class="menu-list book-list"></ul>
             </aside>
           </div>
         </div>
```

Comment:
- [ ] [OPTIONAL] Kindly let's make this container scrollable, since I am not able to see 3 books on the same page, it seems that you have a position absolute that makes it impossible to scroll. You can use the property `overflow: scroll or auto` to make it scroll. 👍